ci/e2e-tests.sh

```diff
@@ -358,6 +358,14 @@ trap "
                         --resource-group "$AZURE_RESOURCE_GROUP_NAME" --name "$common_resource_name" \
                         --query 'location' --output tsv
                 )"
+
+            # `az iot dps create` doesn't have `--tags`, so tag it manually.
```

Comment:
Just fix this comment to be more accurate.

Reply:
I'll replace it with a comment explaining that the tags must be added after the call to `linked-hub create`